etc/containers/systemd/README.rst

```diff
@@ -0,0 +1,72 @@
+Quadlet for Tryton
+==================
+The idea is to launch use quadlets and leverage on systemd's funcionality.
+
+Instructions
+============
+If you want to run these with administrative privileges, you can:
+
+.. code-block:: bash
+
+    # become root
+    su -
+
+    # create the necessary secrets
+    printf 'S0m3 fuck1n6 P4s5w07d.' | podman secret create tryton-postgres-password -
+
+    # copy all these files to /etc
+    cp tryton* /etc/containers/systemd
+
+    # enable tem
+    systemctl daemon-reload
+
+    # start the pod
+    systemctl start tryton-pod
+
+    # check they're running
+    podman ps
+
+    # initialize the DB
+    podman run \
```

Comment:
There's already a script for this here: https://github.com/NorTK/Tryton/blob/master/create_database.sh

Reply:
That script exposes the DB_PASSWORD as an env variable, not a podman secret.

Reply:
Also, the Network entry wouldn't work.

Reply:
Additionally, the `trytond-admin` command is very different. This one set's up the email and asks the user to update the password.